index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Portfolio Site</title>
+		<link rel='stylesheet' type='text/css' href='styles.css'>
+		<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Work+Sans:900&display=swap" rel="stylesheet">
```

Comment:
Normalise CSS file?

Reply:
IE browser check seemed to be fine and I'm not updating any strange properties. I might add at the end of the build